br/pkg/lightning/restore/table_restore.go

```diff
@@ -45,6 +45,8 @@ import (
 	"go.uber.org/zap"
 )
 
+const defaultNewAllocIDCnt = 100000
```

Comment:
Can we sample again for the rest chunk? Instead of using a fixed value. 

Another way may be to record the average row size while reading the chunk.